src/components/Rating.js

```diff
@@ -0,0 +1,36 @@
+import React from 'react'
+import '../components/Rating.css'
+
+
+class Rating extends React.Component {
```

Comment:
It looks good, but you could simply it using a switch statement. Nevertheless, very well!!